.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
               size: { fabric: { minified: fs.statSync('${{ env.minified }}').size, bundled: fs.statSync('${{ env.bundled }}').size } }
             });
       - name: checkout src files
-        run: git checkout origin/master -- src fabric.ts index.ts index.node.ts .browserslistrc rollup.config.mjs
+        run: git checkout ${{ github.event.pull_request.base.sha }}
```

Comment:
not sure why we were checking out only some files. Probably because before dist was in the git history